executor/join.go

```diff
@@ -495,6 +515,8 @@ func (e *HashJoinExec) Next(ctx context.Context, chk *chunk.Chunk) (err error) {
 }
 
 func (e *HashJoinExec) fetchInnerAndBuildHashTable(ctx context.Context) {
+	chkCh := make(chan *chunk.Chunk, e.concurrency)
```

Comment:
1. move this two definition to after the defer func.
2. add a comment for chkCh. (s/ chkCh/ innerResultCh ?)